### PR TITLE
feat(materials): build initial models

### LINKED ISSUE #2: Build Materials classes
https://www.notion.so/StructDes-fd9915fbdcdd45b788b6801f65775a31

### PR DESCRIPTION
**Pull request by mapra99**
Closes #2 

Build some super basic classes for concrete, steel and soil materials. It just defines their constructors with the mechanical properties each of them will accept.

### CODE REVIEW

File: lib/materials/soil.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Materials
+  class Soil < Base
+    attr_reader :bearing_capacity
+
+    def initialize(bearing_capacity:)
+      @bearing_capacity = bearing_capacity
```
Comment (ImMPrada) [inline lib/materials/soil.rb:6]:
```suggestion
      @bearing_capacity = bearing_capacity.to_f
```

File: lib/materials/concrete.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+module Materials
+  class Concrete < Base
+    attr_reader :elastic_module, :design_compression_strength
+
+    def initialize(elastic_module:, design_compression_strength:)
+      @elastic_module = elastic_module
+      @design_compression_strength = design_compression_strength
+    end
```
Comment (ImMPrada) [inline lib/materials/concrete.rb:8]:
```suggestion
    def initialize(elastic_module:, design_compression_strength:)
      @elastic_module = elastic_module.to_f
      @design_compression_strength = design_compression_strength.to_f
    end
```

File: lib/materials/soil.rb (hunk 1 above)
Comment (mapra99) [reply lib/materials/soil.rb]: done!

File: lib/materials/concrete.rb (hunk 2 above)
Comment (mapra99) [reply lib/materials/concrete.rb]: done!

**Review by ImMPrada — APPROVED** [review]
go, go, go!

**Pull request merged**